modules/sdk-coin-apt/src/lib/transaction.ts

```diff
@@ -0,0 +1,16 @@
+import { BaseKey, BaseTransaction } from '@bitgo/sdk-core';
+import { TxData } from './iface';
+
+export class Transaction extends BaseTransaction {
+  canSign(key: BaseKey): boolean {
+    return false;
```

Comment:
should it always be false?

Reply:
By Default in Skeleton PR, its kept as false.
ton : https://github.com/BitGo/BitGoJS/pull/3943/files#diff-1419a46d08351ae9ec9d24697038e25fb70e828a6a00aeb0e79b6574b0a01ccbR6
sui : https://github.com/BitGo/BitGoJS/commit/8ba86b7a10720a14ff1efa9c4616c1f26d27d8e4#diff-f71c74742aabb6007fbb7e7b3586b41105e41530d63ec45bdccd6fc5cb7e1636R6
Handling of actual value would be done in a later PR.